src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -345,6 +345,7 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 			obj.cardCvc.mount( obj.selectors.cardCvc );
 			var cardZipWrapper = document.querySelector( obj.selectors.cardZipWrapper );
 			var zipField = document.createElement( 'input' );
+			zipField.placeholder = 'Zip Code';
```

Comment:
We need to add localization support for this placeholder.

Reply:
@moraleida we should address this before merging.

Reply:
@bordoni @rafsuntaskin this is just a placeholder for the code @codingmusician will be adding to deal w/ those fields. I expect this to be completely reworked in the front end